mesa/model.py

```diff
@@ -24,13 +23,13 @@ def __init__(self, seed=None):
             running: a bool indicating if the model should continue running
 
         """
-        # seed both the numpy and Python random number generators
+        # seed both the numpy random number generators
         if seed is None:
-            self.seed = dt.datetime.now()
+            self.seed = int(dt.datetime.timestamp(dt.datetime.now())) % 39916801
```

Comment:
How about setting `seed` and `random` as class attributes instead?
```
class Model:
    seed =  int(dt.datetime.timestamp(dt.datetime.now())) % 39916801
    random =  numpy.random.RandomState(self.seed)
   
    def __init__(self, seed=None):
        if seed:
            self.seed=seed
            self.random=numpy.random.RandomState(self.seed)
```

This way anyone who subclasses `Model` can set their own instance attribute to override the class attribute. But if they don't want to override, there's still a good default they get without having to call  `super().__init__()` (and thing won't break if people forget to call `super().__init__()`